### PR TITLE
refactor(client): mobile board layout 

### PR DESCRIPTION
**Pull request by Alequez97**
image

### CODE REVIEW

File: GomokuClient/packages/gomoku-core/src/features/Board/Board.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+          ) : (
+            <div
+              key={`${rowIndex}-${colIndex}`}
+              className={styles.tileContainer}
+              onClick={() => onTileClick(rowIndex, colIndex)}
+            />
+          );
+        }),
+      ),
+    [size, tiles, onTileClick],
```
Comment (vkuprin) [inline GomokuClient/packages/gomoku-core/src/features/Board/Board.tsx:36]: useMemo has an unnecessary dependency: 'size'. Either exclude it or remove the dependency array

Comment (Alequez97) [reply GomokuClient/packages/gomoku-core/src/features/Board/Board.tsx]: removed from array

File: GomokuClient/packages/gomoku-core/src/features/Board/Board.module.scss
Comment (vkuprin) [inline GomokuClient/packages/gomoku-core/src/features/Board/Board.module.scss:1]: In this project we use different approach for styling, not css modules, but tailwind with cva. Please see other components

Tailwind, CVA ("class-variance-authority") and tailwind merge with utils from "tailwind-merge" should be used.
Right now current scss implemntation would require as to think about scss BEM approach and lot's of things. I don't want to mess with multiple ways of styling, let's keep consistent. Good examples is: packages/gomoku-core/src/shared/ui/button.tsx

File: GomokuClient/packages/gomoku-core/src/features/Board/Board.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { Meta } from "@storybook/react";
+import { Board } from "./Board";
+
+export default {
+  title: "Components/Board",
+  component: Board,
+  parameters: {
+    layout: "centered",
+  },
+} as Meta<typeof Board>;
```
Comment (vkuprin) [inline GomokuClient/packages/gomoku-core/src/features/Board/Board.stories.tsx:10]: Better would be to use `} as Meta<BoardProps>;`

File: GomokuClient/packages/gomoku-core/src/features/Board/Board.module.scss
Comment (Alequez97) [reply GomokuClient/packages/gomoku-core/src/features/Board/Board.module.scss:1]: Okay, got it. Let's abandon this PR than. Don't think I'll manage to fix it quick

**Review by vkuprin — APPROVED** [review]

**Pull request merged**